Makefile

```diff
@@ -25,10 +25,7 @@ PATCHES += pam_slurm_adopt-send_user_msg-don-t-copy-undefined-d.patch
 PATCHES += pam_slurm_adopt-use-uid-to-determine-whether-root-is.patch
 PATCHES += slurm-rpmlintrc
 else
-SPEC_UPDATE := $(shell cp slurm.spec_centos slurm.spec)
-endif
-ifeq ($(SPEC_UPDATE),)
-# Just used to force a reference.
+$(shell cp slurm.spec_centos slurm.spec)
```

Comment:
I only just thought of this, but you could reduce this down to a single statement with no branching by simply renaming the per-distro specfiles to be suffixed with `$(ID)`.

Reply:
Have two SUSE distros sharing the same spec file, otherwise I would have done something like that.